src/utils/GetChunkSize.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+namespace chunk_size {
+
+/**
+ * Get the chunk size based on the given magic numbers.
+ * @param loop_size The number of iterations.
+ * @param max_num_chunks should be big enough to provide sufficient scheduling, but not too big, as too much overhead
+ * might exist.
```

Comment:
```suggestion
 * @param max_num_chunks There should be enough chunks so that load can be balanced, but not too many since scheduling creates overhead.
```